tools/src/launcher/Main.hs

```diff
@@ -474,38 +483,28 @@ writeWindowsUpdaterRunner runnerPath = liftIO $ do
 
 spawnNode
     :: (FilePath, [Text], Maybe FilePath)
+    -> Bool -- Wallet logging
     -> M (ProcessHandle, Async ExitCode)
-spawnNode (path, args, mbLogPath) = do
+spawnNode (path, args, mbLogPath) shouldLog = do
```

Comment:
`shouldLog` name is confusing. One may think it applies to node (i. e. whether node should log), but it's about wallet (whether wallet should log).